rtl/src/gtest/calls.rs

```diff
@@ -72,6 +72,21 @@ impl GTestRemoting {
         }
         Ok(reply.payload().to_vec())
     }
+
+    fn send_and_get_result(
```

Comment:
Very nit
-> calculate_reply/claclulate_result (similar to how it is called in gear crates)